app/forms/idv/profile_form.rb

```diff
@@ -97,12 +95,5 @@ def parsed_dob
     rescue
       nil
     end
-
-    def result
-      {
-        success: success,
-        errors: errors.messages,
```

Comment:
I don't see any use of analytics here, are we basically throwing away these error messages? Should we be recording these anywhere?